src/libPMacc/include/algorithms/math/doubleMath/trigo.tpp

```diff
@@ -57,6 +57,17 @@ struct Cos<double>
 };
 
 template<>
+struct ACos<double>
+{
+    typedef double result;
+
+    HDINLINE double operator( )(const double& value )
+    {
+        return ::acos( value );
```

Comment:
On the host side `acosf` is not avail in [C++98 only in c99](http://www.cplusplus.com/reference/cmath/acos/). Please use the overloaded function `acos` on the host side.